### PR TITLE
Master wap

### PR DESCRIPTION
**Pull request by zhljen**

### CODE REVIEW

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java [hunk 1]
```diff
@@ -426,29 +423,17 @@ public void deletePartitions(
...
-            filterExpression,
-            partitionNames,
             sortBy,
             sortOrder,
             offset,
             limit,
             includeUserMetadata,
-            includePartitionDetails,
-            includeAuditOnly
+            getPartitionsRequestDto
```
Comment (ajoymajumdar) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: For deletions, the includeAuditOnly should be true always.

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java [hunk 2]
```diff
@@ -595,12 +580,11 @@ public void deletePartitions(
...
             databaseName,
             tableName,
-            filter,
-            null,
             sortBy,
             sortOrder,
             offset,
-            limit
+            limit,
+            new GetPartitionsRequestDto(filter, null, false, false)
```
Comment (ajoymajumdar) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: For deletions, the includeAuditOnly should be true always.

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java [hunk 3]
```diff
@@ -726,22 +709,15 @@ public void deletePartitions(
...
             databaseName,
             tableName,
-            filterExpression,
-            partitionNames,
             sortBy,
             sortOrder,
             offset,
-            limit
+            limit,
+            getPartitionsRequestDto
```
Comment (ajoymajumdar) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: Same comment. For deletions, the includeAuditOnly should be true always.

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java [hunk 4]
```diff
@@ -801,23 +777,16 @@ public void deletePartitions(
...
             tableName,
             viewName,
-            filterExpression,
-            partitionNames,
             sortBy,
             sortOrder,
             offset,
-            limit
+            limit,
+            getPartitionsRequestDto
```
Comment (ajoymajumdar) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: For deletions, the includeAuditOnly should be true always.

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java [hunk 5]
```diff
@@ -877,12 +846,11 @@ public void deletePartitions(
...
             databaseName,
             tableName,
-            filter,
-            null,
             sortBy,
             sortOrder,
             offset,
-            limit
+            limit,
+            new GetPartitionsRequestDto(filter, null, false, false)
```
Comment (ajoymajumdar) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: For deletions, the includeAuditOnly should be true always.

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java [hunk 6]
```diff
@@ -1007,22 +975,15 @@ public void deletePartitions(
...
             databaseName,
             tableName,
-            filterExpression,
-            partitionNames,
             sortBy,
             sortOrder,
             offset,
-            limit
+            limit,
+            getPartitionsRequestDto
```
Comment (ajoymajumdar) [inline metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: For deletions, the includeAuditOnly should be true always.

File: metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java (hunk 1 above)
Comment (zhljen) [reply metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: I github display has issues. This change is for public List<PartitionDto> getPartitionsForRequest(.. 

You can click expanding icon to look at the context.

Comment (ajoymajumdar) [reply metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java]: ok

**Pull request merged**